utilities/src/main/java/io/syndesis/qe/resource/impl/CamelK.java

```diff
@@ -146,27 +145,23 @@ private void installViaBinary() {
                 arguments.append(" --operator-image ").append(TestConfiguration.image(Image.CAMELK));
             }
             arguments.append(" -n ").append(TestConfiguration.openShiftNamespace());
+            arguments.append(" --config ").append(OpenShiftUtils.binary().getOcConfigPath());
 
             final String command = LOCAL_ARCHIVE_EXTRACT_DIRECTORY + "/kamel install" + arguments.toString();
             log.info("Invoking " + command);
-            Runtime.getRuntime().exec(command).waitFor();
+            Process process = Runtime.getRuntime().exec(command);
+            process.waitFor();
+            if (process.exitValue() != 0) {
+                InfraFail.fail("The docker command installing Camel K failed. The exit value is " + process.exitValue() +
```

Comment:
Does the command use docker? If not I would delete docker mention from the log.

Reply:
good point :)